### ISSUE TITLE
Position and window size aren't saved

### ISSUE THREAD

**Issue by sojusnik** [turtl/desktop#12]
On Ubuntu Gnome 15.10 the position and the window size of the main window aren't saved after a restart of Turtl.

Turtl is launched via turtl.desktop in `~/.local/share/applications/` with the following content:
```
[Desktop Entry]
Version=1.0
Name=Turtl
Exec=/home/xxx/yyy/Turtl/turtl
Icon=/home/xxx/yyy/Turtl/turtl.svg
Terminal=false
Type=Application
Categories=Utility;Application;
```

The same is true when launching `/home/xxx/yyy/Turtl/turtl` manually.

Additionally, after launching Turtl, the name of this app is displayed as "unknown" in the OS itself, f.i. in the Gnome-Shell overview and the like.

**Comment by orthecreedence** [comment]
I think there are some things in the desktop app itself I can do to preserve window position/size on close/reopen. I'll put this on the list: https://trello.com/c/Vd9FCfY8

As far as the name being "unknown" is this happening in application menus? In the title of the window itself? It's odd it wouldn't just use the name of the executable.

**Comment by sojusnik** [comment]
Yeah, that's strange indeed, like you'll see [here](http://i.imgur.com/3r4Xg4X.jpg) and [here](http://i.imgur.com/8sVv0iM.png).
